scripts/install/opt/seagate/cortx/motr/sanity/motr_sanity.sh

```diff
@@ -337,7 +351,11 @@ m0spiel_test()
         	libmotr_path=$MOTR_DEVEL_WORKDIR_PATH/motr/.libs/libmotr.so
 	[[ ! -s $libmotr_path ]] && libmotr_path=$libmotr_sys_path
 	format_profile_fid=$(echo $profile_fid | sed 's/.*<\(.*\)>/\1/' | sed 's/:/,/')
-	/usr/bin/m0_filesystem_stats -s $ha_endpoint -p $format_profile_fid -c ${ha_endpoint}000 -l $libmotr_path
+	if [ "$XPRT" = "lnet" ]; then
+		/usr/bin/m0_filesystem_stats -s $ha_endpoint -p $format_profile_fid -c ${ha_endpoint}000 -l $libmotr_path
+	else
+		/usr/bin/m0_filesystem_stats -s $ha_endpoint -p $format_profile_fid -c ${ha_endpoint}0 -l $libmotr_path
+	fi
```

Comment:
If you implement changes recommeded at line 195 and 198 then this if-condition can be removed.

Reply:
The `-c` option here is for the client endpoint address which is not the same as ha endpoint address.
Also, in lnet address format, the tmid value is being changed from 1 to 1000
whereas in libfab address format, we are changing the port number from 2001 to 20010

Reply:
If this is a client end-point address then add a local variable such as client_ep_addr and initialize it accordingly based on the xport protocol. Use this new variable in this command-line so that the code looks easy to understand.

Reply:
Done.